### DISCUSSION TITLE
[alpha][auth] TypeError: provider.isInitialized is not a function

### DISCUSSION THREAD

**Discussion by thesandlord** [firebase/firebase-js-sdk#4492] [Q&A] [answered]
**Environment**
Operating System version: Windows 10
Browser version:  88.0.4324.150
Firebase SDK version: ^0.900.13
Firebase Product: auth (Google sign-in)

I'm trying to initialize the auth object with the default app and its giving me a strange error. Not sure why.

Here is a basic reproduction:

```javascript
import { initializeApp, getApps, getApp } from 'firebase/app'
import { signOut, getAuth } from 'firebase/auth'
const config = require('../config.json')
if (!getApps().length) {
  initializeApp(config)
}

async myFunction(){
   const app = getApp()
   console.log(app) // This works
   const auth = getAuth(app) // This line has the error
   await signOut(auth)
}
myFunction()
```

From the error it seems like it wants a Auth provider, but the documentation for [getAuth](https://modularfirebase.web.app/reference/auth.getauth) only has the app as an argument.

**Comment by davideast** [comment, marked as answer]
Thanks for reporting! This is a recent regression. Right now install the library with the specific version `0.900.12`.

```
npm i firebase@0.900.12 
```

We're working on a new release to fix this so hang tight.

**Reply by Feiyang1** [reply]
0.900.14 was released last week with the fix.
